contrib/com.oneidentity.syslog-ng.plist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<!--
+  launchd job for macOS that runs in the background upon system start.
+  
+  Tested environment: 
+    - macOS Catalina 10.15.3 
+    - syslog-ng 3.25
+    
+  Location to be stored: /Library/LaunchDaemons
+  Permissions: 644 root:wheel
+  If you need to manually start/stop the job:
+    - Start command: sudo launchctl load /Library/LaunchDaemons/com.oneidentity.syslog-ng.plist
+    - Stop command: sudo launchctl unload /Library/LaunchDaemons/com.oneidentity.syslog-ng.plist
+  
+  Author: Roberto Meléndez <@rcmelendez on LinkedIn & Medium>
+  Created: March 11, 2020
+-->
+<dict>
+	<key>Label</key>
+	<string>com.oneidentity.syslog-ng</string>
```

Comment:
Like `com.syslog-ng.plist`? For example, the syslogd implementation of Apple is called: `com.apple.syslogd.plist`

Reply:
Or `com.syslog-ng.syslog-ng.plist`, because ` Label` is a key that uniquely identifies the job. We might want other jobs under `com.syslog-ng.` in the future.